cmd/gosum/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+// DONTCOVER
+
+// nolint: errcheck
+func main() {
+	f, err := os.Open(os.Args[1])
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+
+	h := sha256.New()
+	if _, err := io.Copy(h, f); err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("%x", h.Sum(nil))
+}
```

Comment:
This whole file is probably not necessary. See https://github.com/cosmos/cosmos-sdk/pull/3907/files#r266165301

Reply:
It actually is. We're replacing `sha1sum` with this one.

Reply:
What does it provide? Integrity is guaranteed by the go tools now. 
Maybe I'm not fully understanding the motivation here? Related: https://github.com/cosmos/cosmos-sdk/pull/3907#discussion_r266211832